qemu/tests/cfg/fio_windows.cfg

```diff
@@ -1,16 +1,31 @@
 - fio_windows:
     type = fio_windows
     only Windows
+    images += " stg"
+    image_size_stg = 10G
+    image_name_stg = images/stg
+    create_image_stg = yes
+    force_create_image_stg = yes
     cmd_timeout = 600
-    log_file = "C:\fio_log.txt"
     image_snapshot = yes
+    disk_index = 1
+    disk_letter = I
+    format_disk = yes
     i386, i686:
         install_path = "C:\Program Files\fio"
         install_cmd = 'msiexec /a DRIVE:\fio-x86.msi /qn TARGETDIR="${install_path}"'
     x86_64:
         install_path = "C:\Program Files (x86)\fio"
         install_cmd = 'msiexec /a DRIVE:\fio-x64.msi /qn TARGETDIR="${install_path}"'
+    variants:
+        - with_system_disk:
+            FileName = "C:\fio_system_disk_test.txt"
+            log_file = "C:\fio_log.txt"
+        - with_data_disk:
+            FileName = "I:\fio_data_disk_test.txt"
+            log_file = "I:\fio_log.txt"
     #config_cmd = 'setx -m path "%PATH%;${install_path}\fio;"'
-    fio_cmd = '"${install_path}\fio\fio.exe" --name=fiotest --rw=randrw --iodepth=4 --bs=4k'
-    fio_cmd += ' --size=1G --ioengine=windowsaio --numjobs=4 --runtime=${cmd_timeout} > ${log_file}'
+    fio_cmd = '"${install_path}\fio\fio.exe" --rw=randrw --bs=4k --iodepth=8 --direct=1'
+    fio_cmd += ' --filename="${FileName}" --name=fiotest --ioengine=windowsaio --thread --group_reporting'
+    fio_cmd += ' --numjobs=16 --size=1G --runtime=${cmd_timeout} > ${log_file}'
 
```

Comment:
--size =1G
Does this means the data disk only need about 1G volume?
If yes,why image_size_stg = 10G?